source/slang/hlsl.meta.slang

```diff
@@ -22505,25 +22552,13 @@ struct CoopMat<T : __BuiltinArithmeticType, let S : CoopMatScope, let M : int, l
     [__NoSideEffect]
     Ref<T> __indexRef(int index);
 
+    /// Returns the count as an integer value.
```

Comment:
I feel like we need to have more detailed documentation here explaining the meaning of `count`,
since it doesn't mean the total number of elements in the matrix, instead it refers to the number
of elements that should be processed by a single thread/lane.